### PR TITLE
Remove support for Trollius

### PR DESCRIPTION
**Pull request by bmerry**
Trollius is no longer maintained, and I'm not aware of the support for it being used any more (there are still a few Python 2 users, but they're using the synchronous API).

### CODE REVIEW

File: spead2/send/asyncio.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Generated by bootstrap.sh from spead2/send/asyncio.py. Do not edit.
```
Comment (sratcliffe) [inline spead2/send/asyncio.py]: Isn't this file spead2/send/asyncio.py :)

File: spead2/send/asyncio.py [hunk 2]
```diff
@@ -212,9 +212,12 @@ def callback(arg):
...
             thread (if `comp_vector` is negative).
-        loop : :py:class:`trollius.BaseEventLoop`, optional
-            Event loop to use (defaults to ``trollius.get_event_loop()``)
+        loop : :py:class:`asyncio.BaseEventLoop`, optional
+            Event loop to use (defaults to ``asyncio.get_event_loop()``)
         """
 
 except ImportError:
     pass
+# Monkey-patch asyncio.ensure_future for Python < 3.4.4.
```
Comment (sratcliffe) [inline spead2/send/asyncio.py]: Didn't we drop support for 3.4.x already ?

File: spead2/send/asyncio.py (hunk 1 above)
Comment (bmerry) [reply spead2/send/asyncio.py]: Thanks for catching that - this file was originally generated but is now canonical.

File: spead2/send/asyncio.py (hunk 2 above)
Comment (bmerry) [reply spead2/send/asyncio.py]: Thanks, I missed stripping that bit out.

**Pull request merged**